src/main/app/src/app/zoeken/model/zoek-veld.ts

```diff
@@ -5,7 +5,11 @@
 
 export enum ZoekVeld {
     ALLE = 'ALLE',
-    IDENTIFICATIE = 'IDENTIFICATIE',
+
+    ZAAK_IDENTIFICATIE = 'ZAAK_IDENTIFICATIE',
     ZAAK_OMSCHRIJVING = 'ZAAK_OMSCHRIJVING',
-    ZAAK_TOELICHTING = 'ZAAK_TOELICHTING'
+    ZAAK_TOELICHTING = 'ZAAK_TOELICHTING',
+
+    TAAK_ZAAK_ID = 'TAAK_ZAAK_ID',
```

Comment:
Wat is de TAAK_ZAAK_D? Moet dat niet zijn TAAK_ID?

Reply:
Dit is het zaak id (ZAAK-0000000001) waar de taak onderdeel van is